pages/dkp/kommander/2.2/backup-and-restore/index.md

```diff
@@ -76,15 +80,117 @@ data:
         #   aws_secret_access_key=<REDACTED>
 ```
 
-After you have created the `ConfigMap` with **MinIO**, OR **Amazon S3**, patch the Velero `AppDeployment` by adding the `configOverrides` value. This applies the `ConfigMap` to your instance after the cluster has been configured:
+### Azure Blob
 
-```bash
-cat << EOF | kubectl -n kommander patch appdeployment velero --type="merge" --patch-file=/dev/stdin
-spec:
-  configOverrides:
-    name: velero-overrides
-EOF
-```
+Prerequisites: Create Azure related assets such as storage account, blob containers, resource group, roles, service principals prior to continuing.
+
+1. Prep your credentials-velero file with the values:
+
+    ```bash
+    cat << EOF > ./credentials-velero
+    AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID}
+    AZURE_TENANT_ID=${AZURE_TENANT_ID}
+    AZURE_CLIENT_ID=${AZURE_CLIENT_ID}
+    AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET}
+    AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP}
+    AZURE_CLOUD_NAME=AzurePublicCloud
+    EOF
+    ```
+
+1. Use the credentials-velero file to create the secret (in your case you named it as azure-bsl-credentials). Note that we used --from-env-file referencing the credentials-velero file. This is the change done from the previous instruction.
```

Comment:
This sentenced needs work.

Reply:
the context of this is from the _other_ instruction where `--from-file` is used. we were advised by eng to use `--from-env-file` and tested it. 

we can probably omit the explanation or this sentence since there is no prior context since we did not link the _other_ instruction 

> Note that we used --from-env-file referencing the credentials-velero file. This is the change done from the previous instruction.

Reply:
we can probably remove this: 
> (in your case you named it as azure-bsl-credentials) 

it was a remnant of an instruction given to a customer.